src/preact/vote-buttons.js

```diff
@@ -21,15 +21,8 @@ import {h, Component} from 'preact';
 
 export default class VoteButtons extends Component {
 
-  constructor (props) {
-    super(props);
-
-    this.state = {
-      score: this.props.score
-    };
-
-    this.onVoteUp = this.onVoteUp.bind(this);
-    this.onVoteDown = this.onVoteDown.bind(this);
+  componentWillMount () {
+    this.state.score = this.props.score;
   }
 
```

Comment:
I thought, I’d use `componentWillMount`, because `this.props` is available in it, so the code is simpler. Might be a bad idea.

I’ve also removed the pre-binding and instead bind inside the `onclick` attribute below. This seems fine in simple cases.